packages/react/src/contents/hooks/types/useCommercePages.types.ts

```diff
@@ -2,6 +2,7 @@ import type { CommercePagesRankingStrategy } from '@farfetch/blackout-redux';
 import type { Config, QueryCommercePages } from '@farfetch/blackout-client';
 
 export interface UseCommercePagesOptions extends QueryCommercePages {
+  slug: string;
```

Comment:
This will be a breaking change as well, can't it be at least defined as optional?

Reply:
As mentioned before, this is only used by BG and this will fix the commerce pages in this package version